### DISCUSSION TITLE
Issue Assignee / PR Author as column in Boards view.

### DISCUSSION THREAD

**Discussion by aaronsteers** [community/community#17060] [Projects and Issues]
I expected be able to show each developer's work in a separate column in the issue board. For issues, this is "assigned to". For PRs, the PR author or the "assigned-to" person would be fine. Right now, I can only use custom columns for this split.

Is there any plan to support other fields in the board "columns"?

Thanks!

**Comment by mumrah** [comment]
This seems like a really easy one to add, and would really help our workflow. We use Assignee for the person responsible for reviewing and merging. It's currently hard to use the project board to track our sprints since it's hard to view the PRs by author

**Comment by matchings** [comment]
Please add this 🙏
